app/kubernetes/react/views/yamlReplace/index.tsx

```diff
@@ -14,15 +14,17 @@ export function YAMLReplace({ featureId }: Props) {
       BEFeatureID={featureId}
       message="Applies any changes that you make in the YAML editor by calling the Kubernetes API to patch the relevant resources. Any resource removals or unexpected resource additions that you make in the YAML will be ignored. Note that editing is disabled for resources in namespaces marked as system."
     >
-      <Button
-        type="button"
-        color="warninglight"
-        size="small"
-        onClick={() => {}}
-        disabled
-      >
-        Apply changes
-      </Button>
+      <div className="float-right">
```

Comment:
is this change required for the tooltip ticket?

Reply:
Yes